Rob Wilton #3: clarify "are not ordered" in filter examples (#507)
ietf-wg-jsonpath/draft-ietf-jsonpath-base: draft-ietf-jsonpath-base.md

| `$.arr != $.arr` | false | Array comparison |
| `$.obj == 17` | false | Type mismatch |
| `$.obj != 17` | true | Type mismatch |

> | `$.obj <= $.arr` | false | Objects and arrays do not offer `<` comparison |

Reviewer: Just wondering about the "offering" language. How about the following (and throughout)?
```suggestion
| `$.obj <= $.arr` | false | Objects and arrays cannot be compared using `<` |
```

Author: Well, this sounds like a factual statement, but it is not clear if that is a fundamental property (here, it actually pretty much is), or if it is a result of what we decided.

Author: We could s/offer/provide for/

Reviewer: "provide for" is about the same as "offer" to my taste. How about "do not support"?

Reviewer: (On the factual statement point, "not offering" is equally factual - it doesn't say that this spec doesn't offer.)

Author: The meaning is the same, the connotations aren't.  I'd like stick with "provide for" if you don't like "offer".

Reviewer: It's not a big deal, just a matter of taste. Thanks for considering the alternatives. Merging...
